### PR TITLE
Add methods

### PR DESCRIPTION
**Pull request by looh16**
### _**Add custom methods :**_
**Users**

> A method that returns the 3 most recent posts for a given user.

**Posts**

> A method that updates the posts counter for a user.
> A method which returns the 5 most recent comments for a given post.

**Comments**

> A method that updates the comments counter for a post.
> Likes
> A method that updates the likes counter for a post.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments
   has_many :likes
```
Comment (AnselemOdims) [inline app/models/user.rb]: - [ ] Please include the reference to the foreign key for each of the relationships

Hint
`has_many :posts, foreign_key: 'author_id'`

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @looh16, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/models/user.rb (hunk 1 above)
Comment (looh16) [reply app/models/user.rb]: Thanks!

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @looh16  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes requested by the previous reviewer 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by looh16** [comment]
Thanks!

**Pull request merged**